modules/op-about-whenexpression.adoc

```diff
@@ -0,0 +1,155 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/understanding-openshift-pipelines.adoc
+
+[id="about-whenexpression_{context}"]
+= When expression
+
+When expressions guard task execution by setting criteria for the execution of tasks within a pipeline. They contain a list of components that allows a task to run only when certain criteria are met. When expressions are also supported in the final set of tasks that are specified using the `finally` field in the pipeline YAML file. When expressions in the `finally` task provide efficient guarded execution and improved reusability of tasks.
```

Comment:
Instead of `When expressions in the 'finally' task provide efficient guarded execution and improved reusability of tasks.
` how about `When expressions provide efficient guarded execution of tasks.`

Reply:
Removing this line: When expressions in the `finally` task provide efficient guarded execution and improved reusability of tasks.